postcss.config.js

```diff
@@ -0,0 +1,147 @@
+        'su-bg-lagunita', 
+        'su-bg-lagunita-light', 
+        'su-bg-lagunita-dark', 
+        'su-bg-poppy', 
+        'su-bg-poppy-light', 
+        'su-bg-poppy-dark', 
+        'su-bg-spirited', 
+        'su-bg-spirited-light', 
+        'su-bg-spirited-dark', 
+        'su-bg-illuminating', 
+        'su-bg-illuminating-light', 
+        'su-bg-illuminating-dark', 
+        'su-bg-plum', 
+        'su-bg-plum-light', 
+        'su-bg-plum-dark', 
+        'su-bg-brick', 
+        'su-bg-brick-light', 
+        'su-bg-brick-dark', 
+        'su-bg-archway', 
+        'su-bg-archway-light', 
+        'su-bg-archway-dark', 
+        'su-bg-stone', 
+        'su-bg-stone-light', 
+        'su-bg-stone-dark', 
+        'su-bg-foggy',
+        'su-bg-foggy-light',
+        'su-bg-foggy-dark',
+        'su-text-transparent',
+        'su-text-current', 
+        'su-text-cool-grey', 
+        'su-text-black', 
+        'su-text-black-90', 
+        'su-text-black-80', 
+        'su-text-black-70', 
+        'su-text-black-60', 
+        'su-text-black-50', 
+        'su-text-black-40', 
+        'su-text-black-30', 
+        'su-text-black-20', 
+        'su-text-black-10', 
+        'su-text-black-true', 
+        'su-text-white', 
+        'su-text-cardinal-red', 
+        'su-text-cardinal-red-light', 
+        'su-text-cardinal-red-dark', 
+        'su-text-digital-red', 
+        'su-text-digital-blue', 
+        'su-text-digital-blue-light', 
+        'su-text-digital-blue-dark', 
+        'su-text-digital-green', 
+        'su-text-digital-green-light', 
+        'su-text-digital-green-dark', 
+        'su-text-palo-alto', 
+        'su-text-palo-alto-light', 
+        'su-text-palo-alto-dark', 
+        'su-text-palo-verde', 
+        'su-text-palo-verde-light', 
+        'su-text-palo-verde-dark', 
+        'su-text-olive', 
+        'su-text-olive-light', 
+        'su-text-olive-dark', 
+        'su-text-bay', 
+        'su-text-bay-light', 
+        'su-text-bay-dark', 
+        'su-text-sky', 
+        'su-text-sky-light', 
+        'su-text-sky-dark', 
+        'su-text-lagunita', 
+        'su-text-lagunita-light', 
+        'su-text-lagunita-dark', 
+        'su-text-poppy', 
+        'su-text-poppy-light', 
+        'su-text-poppy-dark', 
+        'su-text-spirited', 
+        'su-text-spirited-light', 
+        'su-text-spirited-dark', 
+        'su-text-illuminating', 
+        'su-text-illuminating-light', 
+        'su-text-illuminating-dark', 
+        'su-text-plum', 
+        'su-text-plum-light', 
+        'su-text-plum-dark', 
+        'su-text-brick', 
+        'su-text-brick-light', 
+        'su-text-brick-dark', 
+        'su-text-archway', 
+        'su-text-archway-light', 
+        'su-text-archway-dark', 
+        'su-text-stone', 
+        'su-text-stone-light', 
+        'su-text-stone-dark', 
+        'su-text-foggy',
+        'su-text-foggy-light',
+        'su-text-foggy-dark'],
```

Comment:
Untestested but perhaps something like this. 

```suggestion
      safelist: {
        deep: [
          /su-bg-[A-Za-z0-9]*(-)?([A-Za-z0-9]*)?/,
          /su-text-[A-Za-z0-9]*(-)?([A-Za-z0-9]*)?/,
        ],
      }
```